package.json

```diff
@@ -14,6 +14,9 @@
     "build:prod:web": "npm run codegen:gql && ng build --prod --base-href=https://CATcher-org.github.io/CATcher/",
     "ng:serve": "ng serve",
     "ng:serve:web": "npm run postinstall:web && ng serve -o",
+    "ng:serve:dev": "ng serve --configuration=dev -o",
+    "ng:serve:test": "ng serve --configuration=test -o",
```

Comment:
We'll need to run `npm run postinstall:web` before `ng serve --configuration=test -o`

Reply:
Sure will add that in 👍